viewer/src/public/RevealManager.ts

```diff
@@ -233,6 +233,14 @@ export class RevealManager<TModelIdentifier> {
     }
   }
 
+  public addUiObject(object: THREE.Group, screenPos: THREE.Vector2, width: number, height: number) {
```

Comment:
`addUiObject` accepts `THREE.Group`, `removeUiObject` accepts `THREE.Object3D`. I think it makes sense to allow `Object3D`. It feels a bit odd that position is `Vector2` while size is two numbers, align types.